contracts/libraries/LibUtils.sol

```diff
@@ -59,6 +59,33 @@ library LibUtils {
         }
     }
 
+    /// @notice Verifies that a signature belongs to a given public key
+    /// @param message the message that was signed
+    /// @param publicKey the public key that was used to sign the message
+    /// @param v signature element
+    /// @param r signature element
+    /// @param s signature element
+    /// @return true if the signature is valid, false otherwise
+    function verifySignature(
```

Comment:
Let's give this a more specific name like `verifyAccusalSignature` or `verifySignatureIsFromPublicKey` if there's a possibility we'll be using this utility elsewhere. We've already got another signature verification utility in this file so things could get confusing